packages/discovery-provider/integration_tests/tasks/test_index_payment_router.py

```diff
@@ -160,6 +171,56 @@ def test_process_payment_router_tx_details_valid_purchase(app):
         assert transaction_record.tx_metadata == str(trackBuyerId)
 
 
+def test_process_payment_router_tx_details_valid_purchase_album(app):
+    tx_response = mock_valid_album_purchase_single_recipient_tx
+    with app.app_context():
+        db = get_db()
+
+    transaction = tx_response.value.transaction.transaction
+
+    tx_sig_str = str(transaction.signatures[0])
+
+    challenge_event_bus = create_autospec(ChallengeEventBus)
+
+    populate_mock_db(db, test_entries)
+
+    with db.scoped_session() as session:
+        process_payment_router_tx_details(
+            session=session,
+            tx_info=tx_response,
+            tx_sig=tx_sig_str,
+            timestamp=datetime.now(),
+            challenge_event_bus=challenge_event_bus,
+        )
+
+        purchase = (
+            session.query(USDCPurchase)
+            .filter(USDCPurchase.signature == tx_sig_str)
+            .first()
+        )
+        assert purchase is not None
+        assert purchase.seller_user_id == 1
```

Comment:
Might recommend using the constants above (`trackOwnerId` etc) to make these assertions clear to future readers. Though that probably also means changing them to `contentOwnerId` and so on.